lms/static/scripts/frontend_apps/components/StudentSelector.tsx

```diff
@@ -53,18 +64,14 @@ export default function StudentSelector({
       )}
     >
       <label
-        className={classnames(
-          'flex-grow font-medium text-sm leading-none',
-          'xl:text-right'
-        )}
+        className="flex-grow font-medium text-sm leading-none xl:text-right"
         data-testid="student-selector-label"
+        htmlFor="student-select"
```

Comment:
I suggest using the `useUniqueId` hook here, on the basis that we should avoid a situation where conflicting IDs could happen if multiple instances of the component were used on the page.